enhancements/microshift/dummy-gateway-interface-ovn.md

```diff
@@ -0,0 +1,458 @@
+  │    ┌─────┼─────────────────┼────────────────────────────┐    │
+  │    │     │                 │       OVN Virtual Topology │    │
+  │    │     │        ┌────────┴────────┐                   │    │
+  │    │     │        │ External Switch │                   │    │
+  │    │     │        └────────┬────────┘                   │    │
+  │    │     │                 │                            │    │
+  │    │     │     ┌───────────┴──────────┐                 │    │
+  │    │     │     │     Gateway Router   │                 │    │
+  │    │     │     └───────────┬──────────┘                 │    │
+  │    │     │                 │                            │    │
+  │    │     │         ┌───────┴────────┐                   │    │
+  │    │     │         │  Join Switch   │                   │    │
+  │    │     │         └───────┬────────┘                   │    │
+  │    │     │                 │                            │    │
+  │    │     │   ┌─────────────┴─────────────┐              │    │
+  │    │     │   │    OVN Cluster Router     │              │    │
+  │    │     │   │        10.244.0.1/24      │              │    │
+  │    │     │   └─────────────┬─────────────┘              │    │
+  │    │     │                 │                            │    │
+  │    │     │        ┌────────┴─────────┐                  │    │
+  │    │     │        │    Node Switch   │                  │    │
+  │    │     │        └────────┬─────────┘                  │    │
+  │    │     │                 │                            │    │
+  │    │  ┌──┴────────────┐    │      ┌──────────────┐      │    │
+  │    │  │ mp0           ├────┴──────┤Pod           │      │    │
+  │    │  │ 10.244.0.2/24 │           │10.244.0.50/24│      │    │
+  │    │  └───────────────┘           └──────────────┘      │    │
+  │    │                                                    │    │
+  │    └────────────────────────────────────────────────────┘    │
+  │                                                              │
+  └──────────────────────────────────────────────────────────────┘
+```
+
+There will be no change to the overlay path taken by the east-west traffic
+between the pods.
+
+The following north-south traffic paths will not be affected either:
+
+* Overlay pod to external
+  * This traffic path is the same as normal local gw mode since the egress
+    traffic will leave OVN-K virtual topology via mp0. Then it will be
+    masqueraded by the host iptables rule and then be sent out from eth0.
+* Host to ClusterIP services backed by overlay pods
+* Host to NodePort services backed by overlay pods
+
+Since we will use a local IP address (10.44.0.0) on br-ex other than the NodeIP
+(10.89.0.3) which is on eth0. OVN-K will treat the NodeIP as an external IP. As
+the endpoint IP of services backed by hostNetwork pods is the NodeIP, and the
+eth0 will not be bound to br-ex. The following traffic cases will be affected:
```

Comment:
nit: not sure I understand how this is expressed.

Reply:
I think what this is saying is that before the change the node's IP was on br-ex, and so it appeared to be an internal IP, but now the node IP is not on br-ex and so will be treated as external. Is that right?

Reply:
I tweak the lines. PTAL